docs/modules/ROOT/pages/accounts.adoc

```diff
@@ -1,632 +1,117 @@
-To relieve smart contract applications such as ERC20 tokens or exchanges from this responsibility, we make use of Account contracts to deal with transaction authentication.
+Instead, the L2 features native account abstraction, and signature validation has to be done at the contract level. To relieve smart contract applications such as ERC20 tokens or exchanges from this responsibility, we make use of Account contracts to deal with transaction authentication.
 
 For a general overview of the account abstraction, see StarkWare's https://medium.com/starkware/starknet-alpha-0-10-0-923007290470[StarkNet Alpha 0.10].
 A more detailed discussion on the topic can be found in https://community.starknet.io/t/starknet-account-abstraction-model-part-1/781[StarkNet Account Abstraction Part 1].
 
+TIP: For detailed information on the usage and implementation check the xref:/api/account.adoc[API Reference] section.
+
 == Table of Contents
 
-* <<quickstart,Quickstart>>
-* <<account_entrypoints,Account entrypoints>>
- ** <<counterfactual_deployments,`Counterfactual deployments`>>
-* <<standard_interface,Standard interface>>
-* <<keys_signatures_and_signers,Keys, signatures and signers>>
-** <<signature_validation, Signature validation>>
- ** <<signer,Signer>>
- ** <<mocksigner_utility,MockSigner utility>>
- ** <<mockethsigner_utility,MockEthSigner utility>>
-* <<call_and_accountcallarray_format,Call and AccountCallArray format>>
- ** <<call,Call>>
- ** <<accountcallarray,AccountCallArray>>
-* <<multicall_transactions,Multicall transactions>>
-* <<api_specification,API Specification>>
- ** <<constructor, `constructor`>>
- ** <<getpublickey,`getPublicKey`>>
- ** <<supportsinterface,`supportsInterface`>>
- ** <<setpublickey,`setPublicKey`>>
- ** <<isvalidsignature,`isValidSignature`>>
- ** <<validate,`\\__validate__`>>
- ** <<validate_declare,`\\__validate_declare__`>>
- ** <<validate_deploy,`\\__validate_deploy__`>>
- ** <<execute,`\\__execute__`>>
-* <<presets,Presets>>
- ** <<account,Account>>
- ** <<eth_account,Eth Account>>
-* <<account_introspection_with_erc165,Account introspection with ERC165>>
-* <<extending_the_account_contract,Extending the Account contract>>
-* <<l1_escape_hatch_mechanism,L1 escape hatch mechanism>>
-* <<paying_for_gas,Paying for gas>>
-
-== Quickstart
-
-The general workflow is:
-
-. Account contract is deployed to StarkNet.
-. Signed transactions can now be sent to the Account contract which validates and executes them.
-
-In Python, this would look as follows:
-
-[,python]
-----
-from starkware.starknet.testing.starknet import Starknet
-from utils import get_contract_class
-from signers import MockSigner
+* <<basic_account,Basic Account>>
+** <<account_entrypoints,Account entrypoints>>
+** <<interface,Interface>>
+* <<standard_account_interface,Standard Account Interface>>
+** <<isrc6_interface,ISRC6 Interface>>
+** <<isrc5_interface,ISRC5 Interface>>
+* <<deploying_an_account,Deploying an Account>>
```

Comment:
After refactoring the erc20 docs, I'm thinking we probably don't need a ToC anymore since it just mirrors the sections in the right column. What do you think?

Reply:
I'm leaning towards removing it as well. Let's wait for @martriay opinion regarding this anyway.